terminal/command.go

```diff
@@ -940,7 +940,7 @@ func printcontext(t *Term, state *api.DebuggerState) error {
 
 	printcontextThread(t, state.CurrentThread)
 
-	if state.CurrentThread.Breakpoint == nil || !state.CurrentThread.Breakpoint.Tracepoint {
+	if state.CurrentThread.Breakpoint == nil || !state.CurrentThread.Breakpoint.Tracepoint || state.CurrentThread.BreakpointInfo == nil {
```

Comment:
This check seem redundant. Can't we assume that if `Breakpoint` is nil, `BreakpointInfo` will also be nil?